scripts/lochness_create_template.py

```diff
@@ -244,31 +250,36 @@ def create_config_template(config_loc: Path, args: object) -> None:
 pii_table: {args.pii_csv}
 lochness_sync_history_csv: {args.lochness_sync_history_csv}
 '''
+
+    if 'rpms' in args.sources:
+        config_example += '''RPMS_PATH: /mnt/prescient/RPMS_incoming
+RPMS_id_colname: src_subject_id
+RPMS_consent_colname: Consent
+'''
+
     if args.s3:
         s3_lines = f'''AWS_BUCKET_NAME: ampscz-dev
 AWS_BUCKET_ROOT: TEST_PHOENIX_ROOT'''
         config_example += s3_lines
     
-    redcap_lines = f'''
-redcap:'''
-
-    config_example += redcap_lines
+    if 'redcap' in args.sources:
+        config_example += '\nredcap:'
 
-    for study in args.studies:
-        redcap_deidentify_lines = f'''
+        for study in args.studies:
+            redcap_deidentify_lines = f'''
     {study}:
         deidentify: True
         data_entry_trigger_csv: {args.det_csv}
         update_metadata: True'''
-        config_example += redcap_deidentify_lines
+            config_example += redcap_deidentify_lines
```

Comment:
Why is the w/ space before?

Reply:
Extra logic in line 265 adds extra indentation now in the update.